lib/routing/router.dart

```diff
@@ -37,7 +38,7 @@ class AppRouter extends _$AppRouter {
           children: [
             AutoRoute(
               page: DashboardTabRoute.page,
-              initial: true,
+              initial: RuntimeArgs.chain == 'testchain' ? true : false,
```

Comment:
Dette antar at vi alltid launcher med det runtime-arget

Reply:
You can read it from the active chain, with the most recent PR I merge

Reply:
this runs before GetIt registers dependencies...